ivadomed/config/config_sctTesting.json

```diff
@@ -105,7 +105,6 @@
             "applied_to": ["im", "gt"],
             "dataset_type": ["training"]
         },
-      "NumpyToTensor": {"applied_to": ["im", "gt"]},
```

Comment:
@charleygros can I just treat this as NumpyToTensor: {} ?

Reply:
Correct, thanks!